inst/include/temperature_component.hpp

```diff
@@ -103,7 +103,7 @@ class TemperatureComponent : public IModelComponent {
     double taudif;           // interior ocean heat uptake time scale, yr
     double tauksl;           // sea-land heat exchange time scale, yr
     double taukls;           // land-sea heat exchange time scale, yr
-    unitval q2co2;           // radiative forcing for atmospheric CO2 doubling (W/m2)
```

Comment:
~Why removal of units?~

Reply:
Oh no it has units and I need to add them!

Reply:
Removed it from a unitval because the math it is used in does not like the unitval, beforehand we had some code that basically extracted the unitval information and reassigned it as a const double which seemed uncessairy.